JeffKersting/blockTracker
# Feature currency selection

#### What does  this PR do?

Adds a currency selection on the dashboard page, allowing users to view crypto in different currencies, with different currency formats.

#### Where should the reviewer start?

Review the currency-dropdown-component and currency-format-component in the currency dropdown folder.

#### Is this a bug fix or a feature?

Feature.

#### How should this be manually tested?

In browser, change the selected currency and/or format and ensure that the currency display is properly changed.
